jekyll/_cci2/deployment-integrations.md

```diff
@@ -257,6 +326,38 @@ workflows:
               only: master
 ```
 
+### Orb Deployment Example
+
+CircleCI has developed a CloudFoundry Orb that you can use to simplify your configuration workflows. The [CircleCI Orbs Registry](https://circleci.com/orbs/registry/) contains several different examples of how you can perform certain tasks with a CloudFoundry, including the example below that shows how you can build and deploy your CloudFoundry application in a single job.
+
+```
+version: 2.1
+
+build_and_push:
+    description: |
+      Build and deploy your application in a single job.
+    usage:
+      version: 2.1
+
+      orbs:
+        cloudfoundry: circleci/cloudfoundry@1.0
+
+      workflows:
+        version: 2
+        build-deploy:
+          jobs:
+            - cloudfoundry/push:
+                appname: your-app
+                org: your-org
+                space: your-space
+                build_steps:
+                  - run: # your build steps
+                  - run: # you can have more, too
+                manifest: # path to manifest.yml file
+                package: # path to application package
+```
+If you would like more detailed information about various CloudFooundry orb examples that you can use in your configurations, refer to the the [CircleCI Orbs Registry](https://circleci.com/orbs/registry/).
```

Comment:
CouldFooundry-> CloudFoundry
I think this link should go directly to the CF orb instead of the top of the Registry

Reply:
Fixed